installer/win/install_instructions_win.md

```diff
@@ -492,3 +492,12 @@ The default (if not specified, or if specified as empty string "") is
 
 
 
+-------------------------------------------------------------------
+### OVERRIDE_USER_AGENT
+
+Expected values:
+
+* "true"
+  "Electron/X.X" is removed from user-agents.
+* "false"
+  User-agents are not modified.
```

Comment:
Add an example of how the command line could look like

Example, install with override user agent true
msiexec /i Symphony.msi OVERRIDE_USER_AGENT="true"